hack/run-e2e-conformance-virtual-ocp.sh

```diff
@@ -204,6 +206,16 @@ podman push --tls-verify=false "${SRIOV_NETWORK_OPERATOR_IMAGE}"
 podman push --tls-verify=false "${SRIOV_NETWORK_CONFIG_DAEMON_IMAGE}"
 podman push --tls-verify=false "${SRIOV_NETWORK_WEBHOOK_IMAGE}"
 
+
```

Comment:
can you please add the network resource injector also?

Reply:
sure